plugin/specs.py

```diff
@@ -0,0 +1,81 @@
+first_level_template = """
+## {id} {title}
+"""
+
+second_level_template = """
+### {id} {title}
+"""
+
+
+third_level_template = """
+#### {id} {title}
+
+This example makes use of {intro} containing the following data:
+
+{input_tables}
+
+`{series}` returns the following patient series:
+
+| patient | value |
+| - | - |
+{output_rows}
+"""
```

Comment:
To match the code that generates the JSON, would it be better to talk about "chapters", "sections", "paragraphs"?

Reply:
Guessing it's that, shout if there's more to do but I'm going to merge now to get the EHR reference page deployed (happy to come back and fix though)